apps/cyberstorm-storybook/stories/components/TextInput.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { TextInput } from "@thunderstore/cyberstorm";
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSkull } from "@fortawesome/free-solid-svg-icons";
+
+const meta = {
+  title: "Cyberstorm/Components/TextInput",
+  component: TextInput,
+} as ComponentMeta<typeof TextInput>;
+
+const defaultArgs = {
+  placeholder: "Search...",
+};
+
+const Template: ComponentStory<typeof TextInput> = (args) => {
+  const [searchValue, setSearchValue] = React.useState("");
+  const [leftIconTriggered, setleftIconTriggered] = React.useState(false);
+  const [rightIconTriggered, setrightIconTriggered] = React.useState(false);
+
+  const leftIconHook = () => {
+    setleftIconTriggered(true);
+  };
+
+  const rightIconHook = () => {
+    setrightIconTriggered(true);
+  };
+
+  const clearInput = () => {
+    setSearchValue("");
+  };
+
+  const onChange = (event) => {
+    setSearchValue(event.target.value);
+  };
+
+  React.useEffect(() => {
```

Comment:
What's the idea here? When `searchValue`'s value changes to "foo", set `searchValue`'s value to "foo"?

Reply:
Woops, remnant of trying out the `useEffect`. Useless code, removing it. 👍